anndata/readwrite/read.py

```diff
@@ -420,18 +420,24 @@ def read_h5ad(filename, backed: Union[bool, str] = False):
         loading it into memory (`memory` mode). `True` and 'r' are
         equivalent. If you want to modify backed attributes of the AnnData
         object, you need to choose 'r+'.
+    chunk_size
+        Used only when loading sparse dataset that is stored as dense.
+        Loading iterates through chunks of the dataset of this row size
+        until it reads the whole dataset.
+        Higher size means higher memory consumption and higher loading speed.
     """
 
     if backed:
         # open in backed-mode
         return AnnData(filename=filename, filemode=backed)
     else:
         # load everything into memory
-        d = _read_h5ad(filename=filename)
+        d = _read_h5ad(filename=filename, chunk_size=chunk_size)
         return AnnData(d)
 
 
-def _read_h5ad(adata: AnnData = None, filename: Optional[PathLike] = None, mode: str = None):
+def _read_h5ad(adata: AnnData = None, filename: Optional[PathLike] = None,
+               mode: str = None, chunk_size: int = 6000):
```

Comment:
could you please format function definitions with long parameter lists like this?

```py
def funcname(
    many_many: params, …
):
    function(body)